### PR TITLE
enable PreferToOverPairSyntax rule for detekt code base

### PR DESCRIPTION
**Pull request by marschwar**
I hope this is not too annoying ;) Just a few left

### CODE REVIEW

File: config/detekt/detekt.yml [hunk 1]
```diff
@@ -188,12 +188,14 @@ style:
     active: true
   MaxLineLength:
     active: true
-    excludes: ['**/test/**', '**/*Test.kt', '**/*Spec.kt']
+    excludes: [ '**/test/**', '**/*Test.kt', '**/*Spec.kt' ]
```
Comment (cortinico) [inline config/detekt/detekt.yml:191]: This is unrelated

**Review by cortinico — APPROVED** [review]

File: config/detekt/detekt.yml [hunk 2]
```diff
@@ -173,7 +173,7 @@ style:
     excludes: ['**/*.kt']
     includes: ['**/detekt-api/src/main/**/api/*.kt']
   MagicNumber:
-    excludes: [ '**/test/**', '**/*Test.kt', '**/*Spec.kt' ]
+    excludes: ['**/test/**', '**/*Test.kt', '**/*Spec.kt']
```
Comment (marschwar) [inline config/detekt/detekt.yml:176]: This is unrelated but consistent with the rest of the file. I finally changed my IDE autoformatter

**Review by BraisGabin — APPROVED** [review]

**Review by schalkms — APPROVED** [review]
